src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static class EnumTests
+    {
+        private static readonly string s_jsonStringEnum =
+                @"{" +
+                @"""MyEnum"" : ""Two""" +
+                @"}";
+
+        private static readonly string s_jsonIntEnum =
+                @"{" +
+                @"""MyEnum"" : 2" +
+                @"}";
+
+        [Fact]
```

Comment:
Could we make these theories with `InlineData` so we can add additional test strings later?

Reply:
Yes, when I extend the tests I will do this to use inlinedata, like I did with the object tests.